### PR TITLE
Add documentation for the USE_GPU option in MaaS_ML

### PR DESCRIPTION
**Pull request by ImenBizid**

### CODE REVIEW

File: src/docs/PML/PMLUserGuide.adoc [hunk 1]
```diff
@@ -478,6 +478,12 @@ Execute the workflow by setting the different workflow's variables as described
 | `MODEL_SERVICE_NODE_NAME`
 | The name of the node where the service will be deployed. If empty, the service will be deployed on an available node selected randomly.
 | String (default=Empty)
+| `USE_NVIDIA_RAPIDS`
+| If True, the service will be configured to use the GPU and the library Nvidia Rapids.
```
Comment (dianajlailaty) [inline src/docs/PML/PMLUserGuide.adoc:482]: and the Nvidia Rapids library

File: src/docs/PML/PMLUserGuide.adoc [hunk 2]
```diff
@@ -488,6 +494,9 @@ Execute the workflow by setting the different workflow's variables as described
 | `ENGINE`
 | Container engine.
 | String (default="$CONTAINER_PLATFORM")
+| `GPU_ENABLED`
+| Configures the service to use the GPU and the library Nvidia Rapids.
```
Comment (dianajlailaty) [inline src/docs/PML/PMLUserGuide.adoc]: same

File: src/docs/PML/PMLUserGuide.adoc [hunk 3]
```diff
@@ -503,6 +512,9 @@ Execute the workflow by setting the different workflow's variables as described
 | `NODE_NAME`
 | It takes by default the value of  `MODEL_SERVICE_NODE_NAME` workflow variable.
 | String (default="$MODEL_SERVICE_NODE_NAME")
+| `GPU_ENABLED`
+| If True, the service will be configured to use the GPU and the library Nvidia Rapids.
```
Comment (dianajlailaty) [inline src/docs/PML/PMLUserGuide.adoc:516]: same

File: src/docs/PML/PMLUserGuide.adoc [hunk 4]
```diff
@@ -654,6 +669,9 @@ Set the following variables:
 | String (default="maas-ml-gui")
 | `ENGINE`
 | Container engine.
+| `GPU_ENABLED`
+| If True, the service will be configured to use the GPU and the library Nvidia Rapids.
```
Comment (dianajlailaty) [inline src/docs/PML/PMLUserGuide.adoc:673]: same

File: src/docs/PML/PMLUserGuide.adoc [hunk 5]
```diff
@@ -670,7 +670,8 @@ Set the following variables:
 | `ENGINE`
 | Container engine.
 | `GPU_ENABLED`
-| If True, the service will be configured to use the GPU and the library Nvidia Rapids.
+| If True, the service will be configured to use the GPU and the library 
+Rapids.
```
[the remark this answers is not in the document]
Comment (ImenBizid) [reply src/docs/PML/PMLUserGuide.adoc]: it is fixed. You are checking an outdated version

**Review by andrewssobral — APPROVED** [review]

**Pull request merged**